dotcom-rendering/src/web/components/FixedMediumSlowXIIMPU.tsx

```diff
@@ -288,35 +288,46 @@ export const FixedMediumSlowXIIMPU = ({
 			return null;
 		}
 		case 1: {
+			/** assert type as we know there is only 1 item */
+			const single = trails as [TrailType];
```

Comment:
We are not relying on the compiler here, but we are in the switch case where there is only one item.

It would be worthwhile to reconsider this logic.